aws-cloudformation-schema/aws-ec2-capacityreservation.json

```diff
@@ -92,11 +95,19 @@
       }
     }
   },
-  "required" : [ "InstanceCount", "AvailabilityZone", "InstancePlatform", "InstanceType" ],
-  "createOnlyProperties" : [ "/properties/Tenancy", "/properties/InstancePlatform", "/properties/InstanceType", "/properties/AvailabilityZone", "/properties/TagSpecifications", "/properties/OutPostArn", "/properties/EphemeralStorage", "/properties/EbsOptimized", "/properties/PlacementGroupArn" ],
+  "required" : [ "InstanceCount", "InstancePlatform", "InstanceType" ],
+  "createOnlyProperties" : [ "/properties/Tenancy", "/properties/InstancePlatform", "/properties/InstanceType", "/properties/AvailabilityZone", "/properties/TagSpecifications", "/properties/OutPostArn", "/properties/EphemeralStorage", "/properties/EbsOptimized", "/properties/PlacementGroupArn", "/properties/AvailabilityZoneId" ],
   "primaryIdentifier" : [ "/properties/Id" ],
   "readOnlyProperties" : [ "/properties/Id", "/properties/AvailableInstanceCount", "/properties/TotalInstanceCount" ],
   "writeOnlyProperties" : [ "/properties/UnusedReservationBillingOwnerId" ],
+  "tagging" : {
```

Comment:
This was added which triggered the new tag type to be registered. I noticed that a lot of resources with this tag structure do not have a tagging block (or not a `tagProperty`) so we might start seeing more of these.